client_plugin/drivers/shared/dockerops/dockerops.go

```diff
@@ -41,7 +43,7 @@ const (
 	// dockerUSocket: Unix socket on which Docker engine is listening
 	dockerUSocket = "unix:///var/run/docker.sock"
 	// Postfix added to names of Samba services for volumes
-	serviceNamePostfix = "fileServer"
```

Comment:
is this working with underscore? I was receiving an error for service name not being DNS compliant while running automation.